src/styles/_reset.scss

```
@@ -0,0 +1,11 @@
+@import "_variables";
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
```

```suggestion
* {
  box-sizing: border-box;
}
```
не нужно тут задавать марджин